pkg/apis/fleet/v1alpha1/types.go

```diff
@@ -248,6 +250,72 @@ type PodSecurityPolicy struct {
 	ValidationFailureAction string `json:"validationFailureAction,omitempty"`
 }
 
+// BackupConfig defines the configuration for the Velero backup engine.
+type BackupConfig struct {
+	// Chart defines the helm chart config of the velero.
+	// default values is
+	//
+	// chart:
+	//   repository: https://vmware-tanzu.github.io/helm-charts
+	//   name: velero
+	//   version: 5.0.2
+	//
+	// +optional
+	Chart *ChartConfig `json:"chart,omitempty"`
+
+	// Storage details where the backup data should be stored.
+	Storage BackupStorage `json:"storage"`
+
+	// VeleroImage Details of the container image to use in the Velero deployment & daemonset
+	// default values is
+	//
+	//  repository: velero/velero
+	//  tag: v1.11.1
+	//  pullPolicy: IfNotPresent
+	//
+	// +optional
+	VeleroImage apiextensionsv1.JSON `json:"veleroImage,omitempty"`
```

Comment:
I think we should use or string type

Reply:
ok

Reply:
We can consider removing this user-specific setting and defaulting to the standard configuration. 

For more advanced settings, the ExtraValues can be utilized as shown below:


```
	// ExtraArgs is the extra chart values setting of backup engine chart.
	//
	// For Example, using following configuration to change image tag or pull policy.
        // extraArgs
	//   image:
	//    repository: velero/velero
	//    tag: v1.11.1
	//    pullPolicy: IfNotPresent
	//
	// +optional
	ExtraArgs apiextensionsv1.JSON `json:"extraArgs,omitempty"`
```

just like https://github.com/kurator-dev/kurator/blob/20fd6ab31961970712cd74909447fb81d0634cba/pkg/apis/fleet/v1alpha1/types.go#L222-L228